helm/templates/ec2nodeclass.yaml

```
@@ -35,3 +41,9 @@ spec:
   amiFamily: Bottlerocket
   amiSelectorTerms:
     - alias: bottlerocket@1.50.0
+  blockDeviceMappings:
+    - deviceName: /dev/xvda
+      ebs:
+        volumeSize: {{ .Values.volumeSize }} # 60Gi
```

Please remove comment:

```suggestion
        volumeSize: {{ .Values.volumeSize }}
```

If someone changes the actual value, this comment will become stale, or they will have to search and replace - yuck!